src/content/language/extension-types.md

```diff
@@ -443,9 +443,10 @@ void main() {
 ```
 
 It's important to be aware of this quality when using extension types,
-and never rely on them in scenarios where the representation type must be concealed. 
-The trade off for using an extension type over a more-secure real object (wrapper class)
-is their lightweight implementation, which can greatly improve performance in some scenarios.
+and never rely on them in scenarios where the representation type must be
+concealed. The trade off for using an extension type over a more-secure real
+object (wrapper class) is their lightweight implementation, which can greatly
+improve performance in some scenarios.
```

Comment:
This text tends to associate extension types with 'reliability' and 'security' (that is: lack of both) in a very vague and broad sense. I think it's more helpful for developers to be reminded about the basic facts, such that they know _how_ they need to be careful. Perhaps we could say something like this?:

```suggestion
and always keep in mind that the extension type exists and matters at
compile time, but it gets erased during compilation. If an expression has
an extension type `E` with representation type `R` as its static type then
the value at run time will be an object with type `R`, and even the type
itself is erased (`List<E>` is exactly the same thing as `List<R>` at run time).
In other words, a real wrapper class can encapsulate a wrapped object
whereas an extension type is just a compile-time view on the wrapped object.
In that sense, the real wrapper is safer; the trade-off is that the
extension type allows us to avoid the wrapper object, which can greatly
improve performance in some scenarios.
```

Reply:
Thanks Erik, this is a much better way to wrap up the page!